R/rhtmlCombinedScatter.R

```diff
@@ -325,12 +325,12 @@ CombinedScatter <- function(
     subtitle.font.color = rgb(44, 44, 44, maxColorValue = 255),
     subtitle.font.family = "Arial",
     subtitle.font.size = 12,
-    subtitle.alignment = "Center",
+    subtitle.alignment = "Center of plot area",
     title = "",
     title.font.color = rgb(44, 44, 44, maxColorValue = 255),
     title.font.family = "Arial",
     title.font.size = 16,
-    title.alignment = "Center",
+    title.alignment = "Center of plot area",
```

Comment:
Can you update the R docs to list the possible values for for `title.alignment`, 'subtitle.alignment`, and `footer.alignment`.

Reply:
Done in https://github.com/Displayr/rhtmlCombinedScatter/pull/50/commits/b4dee8635064f5195df7d99265672441d5f46ece